### DISCUSSION TITLE
My Ram Timing Presets (Version 1.6.1)

### DISCUSSION THREAD

**Discussion by hanai3Bi** [hanai3Bi/Switch-OC-Suite#26] [Ram Timing | Customization]
[Mariko]
Samsung K4U6E3S4AA-MGCL

AUTO_ADJ
2400Mhz
vdd2: 1150
vddq: 620
5-4-4-4-5-5-4

[Erista]
Samsung

AUTO_ADJ
2131Mhz
1175mV
6-2-4-5-3-5-3

**Comment by halop** [comment]
Samsung K4U6E3S4AA-MGCL

AUTO_ADJ
2400Mhz
1125mV
600mV
4-3-4-4-5-5-4

emc_dvb_dvfs_table_t oc_table = { 2400000, { 775, 750, 725, }

![1 60](https://github.com/hanai3Bi/Switch-OC-Suite/assets/4215938/6bfb8868-3503-47cb-a9fd-45efe903b64d)

**Reply by halop** [reply]
Samsung K4U6E3S4AA-MGCL

OC-Suite 1.6.3
CUSTOM_ADJ
2400Mhz
1125mV
600mV
4-4-5-5-5-5-4

emc_dvb_dvfs_table_t oc_table = { 2400000, { 775, 750, 725, }

![1 6 3](https://github.com/hanai3Bi/Switch-OC-Suite/assets/4215938/452f8632-e35c-450d-a9e9-d996d4bebf72)

**Reply by TeleTubby666** [reply]
Strange. I took your settings as reference as I have the same memory in my Aula console but the latencies and bandwidths are significantly worse:

Samsung K4U6E3S4AA-MGCL

CUSTOM_ADJ
2400Mhz
1125mV
600mV
4-4-5-4-5-5-4

![2023063022475200-6463130E01B8A6FA3E1CBB471DEBF81B](https://github.com/hanai3Bi/Switch-OC-Suite/assets/80608448/b5452208-63c5-41d4-a8f2-3aa75ab6141b)
![2023063022492800-6463130E01B8A6FA3E1CBB471DEBF81B](https://github.com/hanai3Bi/Switch-OC-Suite/assets/80608448/fac07f37-bb44-4508-b0d4-0fe753e23e3c)
![2023063022505100-6463130E01B8A6FA3E1CBB471DEBF81B](https://github.com/hanai3Bi/Switch-OC-Suite/assets/80608448/4993ec7a-3cbe-4d6f-abf8-555d5b20a260)

**Reply by halop** [reply]
Still good values, every chip is different,
Maybe you can fine tune it a bit.

**Comment by WE1ZARD** [comment]
Samsung K4U6E3S4AA-MGCL

CUSTOM_ADJ_ALL
UV2
2400mhz
620/1150
6534554

**Reply by halop** [reply]
how much memcpy do you get?

**Reply by WE1ZARD** [reply]
8660.2 mb/s 28102.1 mb/s
86.0 ns /93.0 ns

**Comment by ChanseyIsTheBest** [comment]
RAM Testing
I'm using the same scene from Crash Bandicoot in this here to get fps results
https://gbatemp.net/threads/switch-oc-suite.625066
v1 Erista, Samsung RAM

I feel like I have a bad binning...

AUTO_ADJ_ALL 1175000
0-0-0-0-0-0-0 / 1996800 / 34.5 fps
5-4-4-4-5-4-4 / 1996800 / crash at boot
5-4-4-4-5-5-4 / 1996800 / crash at boot
0-0-0-0-0-0-0 / 2131200 / 38 fps! but crash...
0-0-0-0-0-0-0 / 2099200 / crash in game before fps reading
1-1-1-1-1-1-1 / 2131200 / about same 38fps as all 0s, crash

all 0s and all 1s is identical on AUTO_ADJ_ALL = have same settings

CUSTOM_ADJ_ALL 1175000 
5-4-4-4-5-4-4 / 1996800 / worse perf then 1996 no adjust
5-4-4-4-5-4-4 / 2099200 / slightly better perf then 1996 no adjust mainly stays at 36
6-5-6-5-6-5-6 / 2099200 / always stays at 36, can go to 37 fps, unstable
6-5-6-5-6-5-5 / 2099200 / stays at 37 fps more often, mainly 36 / unstable

NO_ADJ_ALL
1862400 / Default Voltage Stable - Tested for months on end ~34fps
1996800 / 1175000 Stable? ~36fps
1996800 / Default Stable? ~36fps
2099200 / 1175000 Unstable
2133 / 1175000 Unstable

I am in the process of testing stability for NO_ADJ 1996 at default voltage by leaving my Switch on for probably something like at least 12 hours at 1785/921/1996 in a scene that is 80% on 3 CPU cores and 95% on the GPU.

My recommendation for those new to Erista OC is to start at 1862 NO_ADJ_ALL. 
This has been the default for OC Suite used for a long time until recently. It should work for almost anyone.

**Reply by ChanseyIsTheBest** [reply]
OC Suite 1.6.3

[Erista]
Samsung

AUTO_ADJ (Not stable)
2131Mhz
1175mV
6-2-4-5-3-5-3

AUTO_ADJ (Not stable)
2131Mhzz
1175mV
0-0-0-0-0-0-0
35FPS (used to be 38fps)

**Comment by zal16** [comment]
I need to did some changes on timings at this OC Suite version: 
Samsung MGCL  
DRAM Timing: 1 
Vdd2 1150mV, 
vddq 600mv (Default), 
2400Mhz: 5-4-4-4-6-5-3 (6-5-6-5-6-5-6 is the max)

![2023062312574100-0D976DB0FEE9893799D9DD76909258EE](https://github.com/hanai3Bi/Switch-OC-Suite/assets/76079122/433aada4-f80c-4978-8fd7-2b0f548f55bd)

**Comment by EzWiz21** [comment]
v2/Iowa Switch
Micron MT53E512M32D2NP-046 WT:E
CUSTOM_ADJ_ALL
2400MHz
Vdd2: 1175mV, Vddq: 600mV
4-5-6-4-5-5-3 (unstable)
![2023062701412000-9E6A5948F727B745C7EE32B5243A9155](https://github.com/hanai3Bi/Switch-OC-Suite/assets/44066520/29dfb52b-8ac7-4555-a901-760da97174c3)
![2023062701440100-9E6A5948F727B745C7EE32B5243A9155](https://github.com/hanai3Bi/Switch-OC-Suite/assets/44066520/2eedac38-9d7e-44c6-823f-aa6e94501a9c)
![2023062701455000-9E6A5948F727B745C7EE32B5243A9155](https://github.com/hanai3Bi/Switch-OC-Suite/assets/44066520/e0ebc108-36c6-4cbd-90d7-76e26649ef48)

**Reply by hanai3Bi** [reply]
i might be wrong, but isn't E die better die than F die?
try raising SoC voltage
and some timings seems to be too tight. like preset 3 and preset 5

**Reply by EzWiz21** [reply]
I'm not sure, I haven't seen anyone with the F die testing this. Raising Vddq didn't make much difference unless I need to crank it up a little bit more but had to raise Vdd2 to 1200mV for some stability. You're right that the timings are very tight (doing some testing in TOTK and it's definitely not stable xD).

**Reply by hanai3Bi** [reply]
what i meant was this value
https://github.com/hanai3Bi/Switch-OC-Suite/commit/5264cf5ff35250ae6257b6274713cdf01265af9a

for micron ram it's usually F die. haven't seen e die except for yours. mine is also f die. so i think yours have more oc capabilities?

**Reply by EzWiz21** [reply]
CUSTOM_ADJ_ALL
2400MHz
Vdd2: 1200mV, Vddq: 600mV
`emc_dvb_dvfs_table_t oc_table = { 2400000, { 775, 750, 725, } };`
4-5-5-4-4-5-3
![2023062718391900_s](https://github.com/hanai3Bi/Switch-OC-Suite/assets/44066520/21259a6e-852f-434c-901d-beebbc87db82)
Raising SoC voltage did make it run stable in TOTK, 4-5-5-4-5-5-3 was stable too and their memcpy and memset throughput were very close to the initial test (~20MB/s difference).

**Comment by EzWiz21** [comment]
OLED Switch
Hynix H9HCNNNBKMMLXR-NEE
CUSTOM_ADJ_ALL
2400MHz
Vdd2: 1225mV, Vddq: 600mV
`emc_dvb_dvfs_table_t oc_table = { 2400000, { 775, 750, 725, } };`
4-4-5-5-4-5-4
![2023062802223500_s](https://github.com/hanai3Bi/Switch-OC-Suite/assets/44066520/ffc26561-16ff-4b38-b318-01498e474f3c)

**Comment by TeleTubby666** [comment]
This is the best I could achieve while remaining stable in TOTK

OLED Switch
Samsung K4U6E3S4AA-MGCL

CUSTOM_ADJ
2400Mhz
1125mV
600mV
5-5-5-4-5-5-4

![2023070113051100-FC92FBF0157DD4DFBC3526F3EAC1BC4C](https://github.com/hanai3Bi/Switch-OC-Suite/assets/80608448/8ce9f615-8ddb-4323-924a-5c8542ceffbc)
![2023070113063800-FC92FBF0157DD4DFBC3526F3EAC1BC4C](https://github.com/hanai3Bi/Switch-OC-Suite/assets/80608448/a154412f-5bfd-4920-9416-e4204a63e34a)
![2023070113075300-FC92FBF0157DD4DFBC3526F3EAC1BC4C](https://github.com/hanai3Bi/Switch-OC-Suite/assets/80608448/c644ca0f-ddb8-491a-839b-bb0f94a88aee)
![nyx20230702_130712](https://github.com/hanai3Bi/Switch-OC-Suite/assets/80608448/963019b5-bdab-4b22-9576-2fd495e13f13)
